BiblioPixelSerialReceiver.ino

```diff
@@ -7,13 +7,13 @@ https://github.com/ManiacalLabs/BiblioPixelSmartMatrix
 **********************************/
 
 #include "FastLED.h"
-#include <EEPROM.h>
+//#include <EEPROM.h> // not available in M0
```

Comment:
I use EEPROM later in the code below! Cannot remove this. It's used for setting the device ID.
Does the M0 just not have EEPROM? If so, we need to #ifdef around that issue, not remove it.